test/donate.spec.js

```diff
@@ -110,8 +114,33 @@ describe('donations page', function() {
         .elementByCssSelector('#donation-success-container').text()
           .should.become('Thanks for donating $100000 Mary! Please tell a friend! <3')
 
-        .notify(done);
+        .elementByCssSelector('#donations-list > ul > li:nth-child(1)').text()
+          .should.become('Mary - $100000')
+    
+    .notify(done);
+
+    });
+
+    // ZIGGY: I didn't have time to figure out how to test requirejs modules, so
```

Comment:
Haha. Yeah: this whole embeded-inside-of-Ghost thing really paints us into a testing corner. To unit test those methods directly, we'd actually have to run the tests in the browser. On some publicly visible HTML page like whereat.io/test. Have been uninclined to do that thus far, hence all the super top-level integration tests.

Speaking of which, gonna open a TODO Issue to add UI/integration tests to this file for the DonationList (which would, for example, make sure that the first time the user loaded a page, the html contained divs with all the right content, that after they clicked submit, a new div was added to the list with the right content, etc...) Assuming for now you verified manually that it looks okay and merging.